src/Microsoft.AspNet.Mvc.Core/MvcOptions.cs

```diff
@@ -139,11 +139,17 @@ public int MaxModelValidationErrors
 
         /// <summary>
         /// Gets or sets the flag which causes content negotiation to ignore Accept header 
-        /// when it contains the media type */*. <see langword="false"/> by default.
+        /// when it contains the media type */*. <c>false</c>c> by default.
         /// </summary>
         public bool RespectBrowserAcceptHeader { get; set; }
 
         /// <summary>
+        /// Gets or sets a flag to enable content-negotiation to select a formatter based on object type.
```

Comment:
It is the type of the object that is going to be serialized. (Now this type could be based on the declared or runtime type though...didn't want to put that information as its too much detail)